Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.5.3-slim-stretch
+
+WORKDIR /tea-shops
+COPY Gemfile* ./
+
+RUN apk update && apk add --no-cache \
+    build-base \
+    libcurl \
+    libffi-dev \
+    nodejs \
+    imagemagick \
+ && bundle install \
+ && apk del --purge build-base
+
+COPY . ./
+
+ENTRYPOINT ["bundle", "exec"]
```

Comment:
you can create a dummy ruby file with `pust hello world`, just to test if everything is running weel :)